.devcontainer/scripts/debian/register-clang-version.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2025 Contributors to the Media eXchange Layer project.
+# SPDX-License-Identifier: Apache-2.0
+#
+#     --slave /usr/bin/$1 $1 /usr/bin/$1-\${version} \\
```

Comment:
How did those two get in here? Could you please use the opportunity to remove those two lines?